src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs

```diff
@@ -13,6 +13,14 @@
     :aliquot (filter (fn [entity] (= "aliquot" (entity "entityType"))) entities)
     entities))
 
+(defn- create-mock-entities []
+  (map
+    (fn [i]
+      {:entityType (rand-nth ["sample"])
```

Comment:
rand-nth with one item is kinda silly.  Are there other types that we can throw in for mock?